go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/colinmarc/hdfs/v2 v2.2.0
 	github.com/coreos/go-semver v0.3.0
 	github.com/dop251/scsu v0.0.0-20200422003335-8fadfb689669
-	github.com/dropbox/dropbox-sdk-go-unofficial v5.6.0+incompatible
+	github.com/dropbox/dropbox-sdk-go-unofficial v1.0.1-0.20210114204226-41fdcdae8a53
```

Comment:
This go.mod tag looks wrong - the latest version is 5.6 - it has the right commit hash though. Maybe it is just `go mod` doing mad stuff!

https://github.com/dropbox/dropbox-sdk-go-unofficial/tags

Reply:
Yep, blame Go Mod for that! I'll fix it up shortly.

Reply:
@ncw I believe this may be due to the fact that the last GitHub release predates the commit needed, so `go mod` is using the latest commit which will drop the version to v1 for the repo rather than base it off of the release version (if that makes sense). I can't seem to get it to work any other way.

Reply:
Ok! Well it is the right commit version anyway.

I can see the v5.6.0 tag on the `master` branch so it looks like `go mod`.

I wonder if it is worth reporting upstream?